lib/metalsmith.js

```diff
@@ -51,6 +52,16 @@ module.exports = metalsmith(__dirname) // __dirname defined by node.js: name of
     colours
   })
 
+  // rename .md files to .md.njk, so they're passed through the Nunjucks parser
+  .use(renamer({
+    markdown: {
+      pattern: '**/*.md',
+      rename: (name) => {
```

Comment:
@querkmachine Glad you've found [`metalsmith-renamer`](https://github.com/alex-ketch/metalsmith-renamer) and it's still getting updates too 😊 

Think we should use template strings?

```suggestion
      rename: (name) => `${name}.njk`
```